lua/entities/gmod_tardis/modules/sh_falling.lua

```diff
@@ -0,0 +1,191 @@
+            self:EmitSound(snds.FlightLand)
+        end
+
+        if IsValid(self.interior) and TARDIS:GetSetting("flight-internalsound") then
+            local snds_i = self.metadata.Interior.Sounds
+            self.interior:EmitSound(snds_i.FlightLand or snds.FlightLand)
+        end
+    end)
+
+    ENT:OnMessage("fall_crashing_sound", function(self, data, ply)
+        if not TARDIS:GetSetting("sound") then return end
+        local snds = self.metadata.Exterior.Sounds
+
+        if TARDIS:GetSetting("flight-externalsound") then
+            self:EmitSound(snds.FlightFall)
+        end
+
+        if IsValid(self.interior) and TARDIS:GetSetting("flight-internalsound") then
+            local snds_i = self.metadata.Interior.Sounds
+            self.interior:EmitSound(snds_i.FlightFall or snds.FlightFall)
+        end
+    end)
+
+    function ENT:CreateWindSound(time_mult, p)
+        time_mult = time_mult or 0
+        self.wind_sound = self.wind_sound or CreateSound(self, self.metadata.Exterior.Sounds.FlightFallWind)
+        self.wind_sound:SetSoundLevel(90)
+        self.wind_sound:ChangeVolume(time_mult * 0.75 * p / 20)
+        self.wind_sound:Play()
+    end
+
+    function ENT:StopWindSound()
+        if self.wind_sound then
+            self.wind_sound:Stop()
+            self.wind_sound = nil
+        end
+    end
+
+    ENT:AddHook("Think", "wind_sound", function(self)
+        if not self:GetData("free_movement") then
+            self:SetData("free_movement_start", nil)
+        elseif not self:GetData("free_movement_start") then
+            self:SetData("free_movement_start", CurTime())
+        end
+
+        local free_movement_start = self:GetData("free_movement_start")
+        local wind_sound_required = free_movement_start and (CurTime() - free_movement_start > 1)
+
+        if not TARDIS:GetSetting("sound") or not TARDIS:GetSetting("flight-externalsound") or not wind_sound_required then
+            self:StopWindSound()
+            return
+        end
+
+        local time_mult = math.Clamp((CurTime() - free_movement_start - 1) / 5, 0, 1)
+
+        local p=math.Clamp(self:GetVelocity():Length()/125,0,15)
+
+        if not self.wind_sound or not self.wind_sound:IsPlaying() then
+            self:CreateWindSound(time_mult, p)
+            return
+        end
+
+        local ply=LocalPlayer()
+        local e=ply:GetViewEntity()
+        if not IsValid(e) then e=ply end
+
+        if e:EntIndex()==-1 then -- clientside prop
+            local ext=ply:GetTardisData("exterior")
+            e = ext or ply
+        end
+
+        if ply:GetTardisData("exterior")==self and e==self.thpprop and ply:GetTardisData("outside") then
+            self.wind_sound:ChangePitch(95+p,0.1)
+        else
+            local pos = e:GetPos()
+            local spos = self:GetPos()
+            local doppler = (pos:Distance(spos + e:GetVelocity()) - pos:Distance(spos + self:GetVelocity())) / 200
```

Comment:
There still appears to be a popping sound when the sound first starts, you can clearly hear the 1 second delay it makes from it

Reply:
93227df414d7ca11a26aeac9436f037ec7085988
c0909a95dd8ed0a23121336abf97291616ee2303

Reply:
Removed and going to be resolved in a future PR